ecosystem-ci.ts

```diff
@@ -25,16 +25,19 @@ cli
 	.action(async (suites, options: CommandOptions) => {
 		const { root, vitePath, workspace } = await setupEnvironment()
 		const suitesToRun = getSuitesToRun(suites, root)
-		let viteMajor
+		let viteMajor = 5
```

Comment:
this is just a test, i wonder how we should get the viteMajor when we do not have the vite repo since we didn't set it up in commit mode.